skynet/auth/openai.py

```diff
@@ -40,4 +47,13 @@ async def setup_credentials():
 
 
 def get_credentials(customer_id):
-    return credentials.get(customer_id, {}) or {}
+    customer_credentials = credentials.get(customer_id, {}) or {}
+    multiple_credentials = customer_credentials.get('credentialsMap')
+
+    if multiple_credentials:
+        result = [val for val in multiple_credentials.values() if val['enabled'] == True]
```

Comment:
```suggestion
        result = [val for val in multiple_credentials.values() if val['enabled']]
```

Reply:
No need to check explicitly for `True`, unless that thing can have any other truthy values?